docs/matlab_differences.rst

```diff
@@ -0,0 +1,68 @@
+Deliberate Differences from the Original PREP
+=============================================
+
+Although PyPREP aims to be a faithful reimplementaion of the original MATLAB
+version of PREP, there are a few places where PyPREP has deliberately chosen
+to use different defaults than the MATLAB PREP.
+
+To override these differerences, you can set the ``matlab_strict`` argument to
+:class:`pyprep.prep_pipeline.PrepPipeline`, :class:`pyprep.reference.Reference`,
+or :class:`pyprep.find_noisy_channels.NoisyChannels` as ``True`` to match the
+original PREP's internal math.
+
+Differences in RANSAC
+=====================
+
+During the "find-bad-by-RANSAC" step of noisy channel detection, PREP does the
```

Comment:
perhaps you can link to https://pyprep.readthedocs.io/en/latest/generated/pyprep.NoisyChannels.html#pyprep.NoisyChannels

here, using RST. Perhaps:

```suggestion
During the "find-bad-by-RANSAC" step of noisy channel detection (see methods of :class:`pyprep.NoisyChannels`), PREP does the
```

Or perhaps directly link to our functional RANSAC? -> https://pyprep.readthedocs.io/en/latest/generated/ransac.find_bad_by_ransac.html#ransac.find_bad_by_ransac

your call.